templates/rustdoc/navigation.html

```diff
@@ -0,0 +1,248 @@
+                                        {%- endif -%}
+
+                                        {# If the crate is hosted on GitHub, show some stats #}
+                                        {%- if krate.github -%}
+                                            <li class="pure-menu-item">
+                                                <a href="{{ krate.repository_url }}" class="pure-menu-link">
+                                                    <i class="fa fa-github fa-fw"></i>
+                                                    <i class="fa fa-star-o fa-fw"></i> {{ krate.github_stars }}
+                                                    <i class="fa fa-code-fork fa-fw"></i> {{ krate.github_forks }}
+                                                    <i class="fa fa-exclamation-circle fa-fw"></i> {{ krate.github_issues }}
+                                                </a>
+                                            </li>
+
+                                        {# If all the crate has is a repo url, show it #}
+                                        {%- elif repository_url -%}
+                                            <li class="pure-menu-item">
+                                                <a href="{{ krate.repository_url }}" class="pure-menu-link">
+                                                    <i class="fa fa-code-fork fa-fw"></i> Repository
+                                                </a>
+                                            </li>
+                                        {%- endif -%}
+
+                                        <li class="pure-menu-item">
+                                            <a href="https://crates.io/crates/{{ krate.name }}" class="pure-menu-link" title="See {{ krate.name }} in crates.io">
+                                                <i class="fa fa-cube fa-fw"></i> Crates.io
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+
+                                {# Show the crate authors #}
+                                <div class="pure-u-1-2">
+                                    <ul class="pure-menu-list">
+                                        <li class="pure-menu-heading">Authors</li>
+
+                                        {%- for author in krate.authors -%}
+                                            <li class="pure-menu-item">
+                                                <a href="/releases/{{ author[1] }}" class="pure-menu-link">
+                                                    <i class="fa fa-fw fa-user"></i> {{ author[0] }}
+                                                </a>
+                                            </li>
+                                        {%- endfor -%}
+                                    </ul>
+                                </div>
+                            </div>
```

Comment:
> > This has shown up twice now (I think the previous time was on rustdoc pages). Can we split it into a separate template?

> I'm not sure where a previous one is, this is the only instance I can find (the rustdoc navigation bar)

The other place is in `crate/details.html`: https://github.com/rust-lang/docs.rs/pull/879/files#diff-2550c2ab5e8909adfed2ffb04e746370R19